.buildkite/pipeline.elastic-agent-package.yml

```diff
@@ -105,6 +105,58 @@ steps:
         artifact_paths:
           - "build/distributions/**/*"
 
+      - label: ":package: Cross Building and package FIPS linux/amd64 elastic-agent"
+        key: package-elastic-agent_fips
+        agents:
+          provider: "gcp"
+          machineType: "c2-standard-16"
+          diskSizeGb: 400
+        env:
+          PLATFORMS: "linux/amd64"
+          FIPS: true
+        command: |
+          if [[ -z "$${MANIFEST_URL}" ]]; then
+            export MANIFEST_URL=$(buildkite-agent meta-data get MANIFEST_URL --default "")
+            if [[ -z "$${MANIFEST_URL}" ]]; then
+              echo ":broken_heart: Missing MANIFEST_URL variable or empty string provided"
+              exit 1
+            fi
+          fi
+          if [[ -z "$${MAGEFILE_VERBOSE}" ]]; then
+            export MAGEFILE_VERBOSE=$(buildkite-agent meta-data get MAGEFILE_VERBOSE --default "0")
+          fi
+          .buildkite/scripts/steps/package.sh
+        artifact_paths:
+          - "build/distributions/**/*"
+      - label: ":package: Package FIPS linux/arm64 elastic-agent"
+        key: package-elastic-agent-arm-fips
+        agents:
+          provider: "aws"
+          instanceType: "t4g.2xlarge"
+          imagePrefix: "core-ubuntu-2204-aarch64"
+          diskSizeGb: 400
+        env:
+          PLATFORMS: "linux/arm64"
+          PACKAGES: "docker,tar.gz"
+        command: |
+          echo "Add support for multiarch"
+          docker run --privileged --rm tonistiigi/binfmt:master --install all
```

Comment:
I see the step `label: ":package: Package ARM elastic-agent"` also include this, wonder if we could use a script to centralise it, so we avoid duplication here, what do you think?

Reply:
Same as above.

Reply:
pushed a new commit with matrix implementation 3ecfcae5ceeccde2b658921d322de585b0624b81